org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/format/experimental/XMLFormatterExperimentalIndentTest.java

```diff
@@ -31,9 +31,10 @@ public class XMLFormatterExperimentalIndentTest extends AbstractCacheBasedTest {
 	public void startWithSpaces() throws BadLocationException {
 		String content = "\r\n    " + //
 				"   <a></a>";
-		String expected = "<a></a>";
+		String expected = "\r\n" + //

```

Comment:
I think we should keep the current behavior, in other words remove all spaces and line separator before the first start tag element like IJ does.